examples/next/application-headers.yml

```diff
@@ -0,0 +1,82 @@
+asyncapi: '2.0.0'
+id: 'urn:com:smartylighting:streetlights:server'
+info:
+  title: Application Headers example
+  version: '1.0.0'
+  description: A cut of the Streetlights API to test application header changes supporting #112
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  - url: api.streetlights.smartylighting.com:{port}
+    scheme: mqtt
+    description: Test broker
+    baseChannel: smartylighting/streetlights/1/0
+    variables:
+      port:
+        description: Secure connection (TLS) is available through port 8883.
+        default: '1883'
+        enum:
+          - '1883'
+          - '8883'
+
+defaultContentType: application/json
+
+channels:
+  event/{streetlightId}/lighting/measured:
+    parameters:
+      - $ref: '#/components/parameters/streetlightId'
+    subscribe:
+      summary: Receive information about environmental lighting conditions of a particular streetlight.
+      operationId: receiveLightMeasurement
+      message:
+        $ref: '#/components/messages/lightMeasured'
+
+components:
+  messages:
+    lightMeasured:
+      name: lightMeasured
+      title: Light measured
+      summary: Inform about environmental lighting conditions for a particular streetlight.
+      correlationId:
+        location: "$message.header#/MQMD/CorrelId"
+      contentType: application/json
+      headers:
+        MQMD:
+          type: object
+          properties:
+            CorrelId:
+              type: string
+              minLength: 24
+              maxLength: 24
+              format: binary
+        replyTo:
```

Comment:
Isn't replyTo a protocol header? Also, the only server defined above is MQTT and you're defining WebSphere headers. It may deserve a different example document.

Reply:
Not universally but as this is an example will change and come up with a better example